algebra/tests/quat/various.c

```diff
@@ -815,3 +834,110 @@ TEST_GROUP_RUNNER(group_quat_normalize)
 	RUN_TEST_CASE(group_quat_normalize, quat_normalize_std);
 	RUN_TEST_CASE(group_quat_normalize, quat_normalize_biggerValues);
 }
+
+
+/* ##############################################################################
+ * ---------------------        quat_uvec2uvec tests       ----------------------
+ * ############################################################################## */
+
+
+TEST_GROUP(group_quat_uvec2uvec);
+
+
+TEST_SETUP(group_quat_uvec2uvec)
+{
+}
+
+
+TEST_TEAR_DOWN(group_quat_uvec2uvec)
+{
+}
+
+
+TEST(group_quat_uvec2uvec, quat_uvec2uvec_std)
+{
+	quat_t q;
+	vec_t v1 = V1, v2 = V1rotQ8;
+	vec_normalize(&v1);
+	vec_normalize(&v2);
```

Comment:
```suggestion
	quat_t q;
	vec_t v1 = V1;
	vec_t v2 = V1rotQ8;

	vec_normalize(&v1);
	vec_normalize(&v2);
```
This is more readable, plus a split between declarations and function body is necessary. Applies in more tests.